appointment.go

```diff
@@ -40,11 +40,14 @@ type AppointmentService struct {
 }
 
 type AppointmentCreate struct {
```

Comment:
Just to confirm, the API works as expected when sending an explicit `null` value for `description` and `service_location`?

Reply:
Yep, that creates an appointment with an empty string `description` and a nil `service_location` in the response back 👍